fastembed/embedding.py

```diff
@@ -200,17 +221,20 @@ def encode(self, documents: List[str], batch_size: int = 256) -> List[np.ndarray
             )
             # TODO: Should we normalize after all batches are done?
             embeddings = normalize(embeddings).astype(np.float32)
-            yield embeddings
+            yield from embeddings
```

Comment:
If I remember correctly, but I might be wrong — this'd mean we've to change how the integration consumes this as well. Was this handled?

Reply:
Might be, but the version is pinned so we will see if tests are broken or not before releasing. And this API is not exposed to users directly